.github/scripts/latest_version_release.sh

```diff
@@ -28,47 +26,40 @@ pushd "$GITHUB_WORKSPACE/charts/$chart_dir"
     chart_version=$(grep '^version:[[:space:]][[:digit:]]' Chart.yaml | awk '{print $2}') || die "Failed to retrieve current chart version"
     bump_version "${chart_version}" "${major_minor_patch}"
     new_chart_version=$new_version && echo "The new Chart version is: ${new_chart_version}"
-    sed -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
+    sed  -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
 
     # edit app version
-    sed -i "s/appVersion.*/appVersion: ${app_version}/g" Chart.yaml
-    
-    if [[ -z "${sra_chart}" ]]; then
-        git add -A && git commit -m "Updated ${service} helm chart version to latest: ${app_version}" || die "Failed to commit changes to git"
-        git push origin HEAD
-    fi
-popd
+    if [[ "${chart_dir}" == "akeyless-secure-remote-access" ]]; then
+      # edit app service version
+      if [[ "${service}" == "ssh-proxy" ]]; then
+          sra_inner_chart="sshVersion"
+      elif [[ "${service}" == "zero-trust-bastion" ]]; then
+          sra_inner_chart="ztbVersion"
+      else
+          die "Bad SRA service name"
+      fi
+
+      sed -i "s/${sra_inner_chart}.*/${sra_inner_chart}: ${app_version}/g" Chart.yaml
+      # edit sra app version
+      ztb_app_ver=$(grep 'ztbVersion' Chart.yaml | awk '{print $2}')
+      ssh_app_ver=$(grep 'sshVersion' Chart.yaml | awk '{print $2}')
+      sed -i "s/appVersion.*/appVersion: ${ztb_app_ver}_${ssh_app_ver}/g" Chart.yaml
 
-if [[ -n "${sra_chart}" ]]; then
-    pushd "$GITHUB_WORKSPACE/charts/$sra_chart"
-        # bump chart version
-        chart_version=$(grep '^version:[[:space:]][[:digit:]]' Chart.yaml | awk '{print $2}') || die "Failed to retrieve current chart version"
-        bump_version "${chart_version}" "${major_minor_patch}"
-        new_chart_version=$new_version && echo "The new Chart version is: ${new_chart_version}"
-        sed -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
+      git add -A && git commit -m "Updated ${service} helm chart version to latest: ${app_version}" || die "Failed to commit changes to git"
+      git push origin HEAD
```

Comment:
This approach is very fragile, you must do `git pull` before committing anything 
Moreover, the push origin will only work if the branch exist on the remote origin

Reply:
remember this run on GitHub action, so you have a checkout action before this. in addition this workflow is triggered from other repo, that is also checks out the code

Reply:
Sure, but what if someone marge something before the action gets around to doing its thing?

Reply:
There is a previous step on the workflow:

`      - name: Fetch history
        run: git fetch --prune --unshallow`
        
Those this cover the issue you talk about?

Reply:
updated